src/legacy/core_plugins/interpreter/public/load_browser_plugins.js

```diff
@@ -36,7 +38,7 @@ function addFunction(fnDef) {
 
 functions.forEach(addFunction);
 
-createSocket(basePath).then(async () => {
-  await populateBrowserRegistries(types, basePath);
-  await initializeInterpreter();
+loadBrowserRegistries(types, basePath).then(async () => {
+  const socket = await createSocket(basePath, functionsRegistry);
```

Comment:
Looks like this is also going to close https://github.com/elastic/kibana/issues/27175 too 👍